[fix] youtube video, audio, playback speed are not working correctly

saeedbashir opened the pull request:
### Description

[LEARNER-8612](https://openedx.atlassian.net/browse/LEARNER-8612)

I've updated the version of youtube library `YoutubePlayer-in-WKWebView` from `0.3.5` to `0.3.8` and made some changes to the js player vars. Now everything(audio, video, playback, fullscreen) is working fine.

While updating the youtube library following libraries has also been updated:

1. FirebaseABTesting (`8.0.0`  ->  `8.9.0`)
2. FirebaseCoreDiagnostics (`8.0.0`  ->  `8.9.0`)
3. FirebaseInstallations (`8.0.0`  ->  `8.9.0`)
4. FirebaseRemoteConfig (`8.0.0`  ->  `8.9.0`)
5. GoogleDataTransport (`9.0`  ->  `9.1`)
8. PromisesObjC (`1.2`  ->  `< 3.0, >= 1.2`) 
9. GoogleUtilities/* (  ->  `7.6`)
10. GTMSessionFetcher/Core (`1.5.0` -> `1.7.0`)
11. Protobuf (`3.17.0` -> `3.19.1`)
12. SDWebImage (`5.11.1` -> `5.12.1`)



### How to test this PR

Go the course videos and find out a unit with youtube videos. And make sure everything (audio, video, playback, screen) is working as expected.

colinbrash reviewed (approved):
LGTM!

mumer92 commented:
@saeedbashir 
Other libraries have been also updated, can you please update description in PR and on the ticket as well?

saeedbashir commented:
> @saeedbashir Other libraries have been also updated, can you please update description in PR and on the ticket as well?

@mumer92 I've updated the PR description. As most of the libraries are dependency libraries and updated because of updating the pod repos so there isn't any need to mention them on the ticket.
